packages/frontend/src/components/VisualizationConfigs/TableConfigPanel/GeneralSettings.tsx

```diff
@@ -248,6 +250,25 @@ const GeneralSettings: FC = () => {
                         setShowResultsTotal(!showResultsTotal);
                     }}
                 />
+                <Tooltip
+                    disabled={!canUsePivotTable}
```

Comment:
I think the variable name is misleading. We should either introduce another variable that better describes whether the pivot table is enabled or just rename this one to `isPivotTableEnabled` or something similar.